src/ffi/wrapper.hpp

```diff
@@ -1,5 +1,6 @@
 // header wrapper to feed into bindgen
 
+#define QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
Is there a way to make this opt in at the rust layer? We don't want to expose this stuff by default, since it can change.

Reply:
Not in the rust FFI layer. Once the code is generated, all apis are there regardless of preview of not.
Or we can generate the code twice with 1 copy containing preview apis and 1 without, but I prefer not to do this since it is a burden to maintain and regenerate.
We should keep using the preview feature flag in the wrapper layer that macro out the preview apis.